frontstage/templates/surveys/surveys-add.html

```diff
@@ -68,11 +104,17 @@ <h1 class="u-fs-xl ons-u-mt-s">Add a survey</h1>
                             })
                         }}
                     </div>
-                    <div class="ons-grid__col ons-u-ml-m">
-                        <a href="{{ url_for('surveys_bp.get_survey_list', tag='todo') }}" id="cancel_button" role="button">Cancel</a>
-                    </div>
                 </div>
         </form>
+        {%
+            call onsDetails({
+                "id": "details-example-with-warning",
+                "title": "Need help with the enrolment code?"
+            })
+        %}
+            <p>If your enrolment code has been lost, does not work or you did not receive one, request a new code.</p>
+            <p>To do this, call +44 300 1234 931 or <a href='/surveys/help/something-else'>send a message</a></p>
```

Comment:
I have spoken to Karen about where specifically this "send a message" link needs to point to. It will eventually point to the link created as part of the "Send message" journey. I have made a note in both cards and links so that this link is updated once the work has been completed.

Reply:
If you go down this route currently it leads to 500 errors but as the journey is updated in a separate card no issues. Just calling it out for the following card associated with adding a survey.